### PR TITLE
preview create job

### PR DESCRIPTION
**Pull request by lamphan95**
#### What's this PR do?
preview when create job
#### What are the relevant tickets?
resolved #ticket_ID

#### What are the features affected?
none

### CODE REVIEW

File: app/views/employers/jobs/_form.html.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <%= simple_form_for [:employers, @job], remote: true do |f| %>
     <%= f.hidden_field :company_id %>
     <%= f.input :name, label: t(".job_name"), input_html: {class: "job-create-name"} %>
-    <%= f.input :description, label: t(".description"), as: :ckeditor %>
+    <div><%= f.input :description, label: t(".description"), as: :ckeditor %></div>
```
Comment (lamphan95) [inline app/views/employers/jobs/_form.html.erb]: .

File: app/views/employers/jobs/_form.html.erb [hunk 2]
```diff
@@ -58,11 +58,170 @@
       <%= render partial: "add_survey", locals: {f: f} %>
     <% end %>
     <center>
-      <div class="centerize create-job-btn">
-        <%= f.button :submit, label: t("jobs.form.save_job"),
-          class: "btn btn-primary btn-sm w-105 m-top-20" %>
+      <div class="row">
+      <%= f.button :submit, label: t("jobs.form.save_job"),
```
Comment (lamphan95) [inline app/views/employers/jobs/_form.html.erb]: .

File: app/assets/javascripts/employers/preview.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
+$(document).ready(function(){
+  $('#btn-preview').click(function(){
+    $('.job-form').hide(700);
+    $('#preview').show(700);
+
+    job = getDataForm('new_job');
+
+    $('#job-expried').text(job.end_time)
+    $('.job-title').text(job.name);
+    $('.branch-name').text(`(branch in ${job.branch})`.toUpperCase());
```
Comment (VuThiTranVan) [inline app/assets/javascripts/employers/preview.js]: I18n `branch in `

File: app/assets/javascripts/employers/preview.js [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+      job.currency = $(`#job_currency_id option[value=${$('#job_currency_id').val()}]`).text();
+    }
+    return job
+  }
+
+  function convertSalary(min_salary, max_salary, currency) {
+    if (max_salary === '') {
+      return `(Up To ${min_salary} ${currency})`
+    } else {
+      return `(${min_salary} ${currency} - ${max_salary} ${currency})`
```
Comment (hhhhuy112) [inline app/assets/javascripts/employers/preview.js]: (tat) Mày muốn anh chết nữa hả

File: app/assets/javascripts/employers/preview.js [hunk 5]
```diff
@@ -0,0 +1,189 @@
...
+
+    if ($('#job_currency_id').val() !== '') {
+      job.currency = $(`#job_currency_id option[value=${$('#job_currency_id').val()}]`).text();
+    }
+    return job
+  }
+
+  function convertSalary(min_salary, max_salary, currency) {
+    if (max_salary === '') {
+      return `(Up To ${min_salary} ${currency})`
```
Comment (hhhhuy112) [inline app/assets/javascripts/employers/preview.js]: (tat) Mày muốn anh chết nữa hả

**Pull request merged**